docs/book/src/multiversion-tutorial/testdata/project/internal/controller/cronjob_controller_test.go

```diff
@@ -129,7 +129,7 @@ var _ = Describe("CronJob controller", func() {
 			By("By checking the CronJob has zero active Jobs")
 			Consistently(func(g Gomega) {
 				g.Expect(k8sClient.Get(ctx, cronjobLookupKey, createdCronjob)).To(Succeed())
-				g.Expect(createdCronjob.Status.Active).To(HaveLen(0))
+				g.Expect(createdCronjob.Status.Active).To(BeEmpty())
```

Comment:
Hey, great work! 🎉

This looks really good, and you’ve already figured out what needs to be changed. 👏

Just a quick note:
All the samples under the docs (inside the book directory) are generated automatically when we run make generate. This command updates both the samples in testdata and the ones in docs because it calls make generate-docs.

So, instead of modifying the files directly in docs, the changes should be made in hack/docs first. After that, you’ll need to run:

sh
Copy
Edit
make generate-docs
to generate the final result correctly.

Could you take a look and check if you can apply the changes this way? It seems like you might be able to streamline some of the modifications as well.

Great job overall! 🚀 Looking forward to your update.

Reply:
Thanks a lot for the clarification 🙏 . Will reflect that on the PR accordingly.

Reply:
If you run make generate-docs locally, you will see that all your changes here will be lost.
If we allow triggering the CI, you will see that it fails because we check to ensure that the changes are always properly generated.